src/classes/__tests__/MyHandler.ts

```diff
@@ -0,0 +1,16 @@
+import MyHandler from '../MyHandler';
+import Bot from '../Bot';
+import BotManager from '../BotManager';
+import SteamID from 'steamid';
+import SteamUser from 'steam-user';
+
+jest.mock('SteamUser');
+
+test('onMessage', () => {
+    SteamUser.chatMessage.mockResolvedValue();
```

Comment:
maybe you should add `const client: SteamUser = new SteamUser()` before this line and replace `SteamUser.chatMessage` with `client.chatMessage` then let see if this check failure disappears.